coredump/criu_coredump/coredump.py

```diff
@@ -794,7 +794,11 @@ def _get_page(self, pid, page_no):
         off = 0  # in pages
         for m in pagemap[1:]:
             found = False
-            num_pages = m.get("nr_pages", m.compat_nr_pages)
+            num_pages = m.get("nr_pages", m.get("compat_nr_pages"))
+
+            if num_pages is None:
+                raise ValueError("num_pages and compat_nr_pages are missing")
```

Comment:
how is it possible? cmpat_nr_pages is a required field: https://github.com/checkpoint-restore/criu/blob/e879d69a01bfd402b85947b06fef354ccd243e54/images/pagemap.proto#L13

Reply:
should we use m['compat_nr_pages']?